### PR TITLE
feat: update charts

### PR DESCRIPTION
**Pull request by nabeelsaabna**
CE helm chart
- add bitbucket variables
- add postgres variables
- add imagePullSecrets option

EE helm chart
- remove the postgres deployment
- add bitbucket variables
- add imagePullSecrets option
- fix: move variables from worker to server

### CODE REVIEW

File: helm-charts/mend-renovate-ee/templates/server-deployment.yaml [hunk 1]
```diff
@@ -171,19 +211,23 @@ spec:
...
           readinessProbe:
-            {{- toYaml . | nindent 12 }}
+          {{- toYaml . | nindent 12 }}
           {{- end }}
           resources:
-            {{- toYaml .Values.renovateServer.resources | nindent 12 }}
+          {{- toYaml .Values.renovateServer.resources | nindent 12 }}
           volumeMounts:
-            - name: {{ .Release.Name }}-database-volume
+            - name: { { .Release.Name } }-database-volume
```
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply helm-charts/mend-renovate-ee/templates/server-deployment.yaml:219]: all other places it's `{{ .Release.Name }}` without spaces
spaces maybe a bug, but regardless i wanted to make it consistent

File: helm-charts/mend-renovate-ee/values.yaml [hunk 2]
```diff
@@ -17,20 +17,14 @@ license:
   existingSecret:
 
 postgresql:
-    # if to use PostgreSQL, if set to 'false' SQLite will be used instead
+    # if to use PostgreSQL, if unset SQLite will be used instead
```
[the remark this answers is not in the document]
Comment (nabeelsaabna) [reply helm-charts/mend-renovate-ee/values.yaml]: done 👍

File: helm-charts/mend-renovate-ce/values.yaml [hunk 3]
```diff
@@ -20,7 +20,8 @@ renovate:
   # Please allow up to 24 hours to receive your license key by email.
   mendRnvLicenseKey:
 
-  # Which platform Mend Renovate will connect to. For example "github" or "gitlab"
+  # Which platform Mend Renovate will connect to.
+  # valid values: "github", "gitlab" or "bitbucket-server"
```
Comment (viceice) [inline helm-charts/mend-renovate-ce/values.yaml:24]: should be sorted alphabetically

File: helm-charts/mend-renovate-ee/values.yaml [hunk 4]
```diff
@@ -43,7 +37,8 @@ renovateServer:
   # Additional server env vars
   extraEnvVars: []
 
-  # Which platform Mend Renovate will connect to. For example "github" or "gitlab"
+  # Which platform Mend Renovate will connect to.
+  # valid values: "github", "gitlab" or "bitbucket-server"
```
Comment (viceice) [inline helm-charts/mend-renovate-ee/values.yaml:41]: Same here.

**Review by viceice — COMMENTED** [review]
you should use a helm readme generator to automatically update the readme from values.

- https://github.com/bitnami/readme-generator-for-helm

File: helm-charts/mend-renovate-ce/values.yaml (hunk 3 above)
Comment (nabeelsaabna) [reply helm-charts/mend-renovate-ce/values.yaml:24]: done

File: helm-charts/mend-renovate-ee/values.yaml (hunk 4 above)
Comment (nabeelsaabna) [reply helm-charts/mend-renovate-ee/values.yaml]: done

**Comment by nabeelsaabna** [comment]
> you should use a helm readme generator to automatically update the readme from values.
> 
> * https://github.com/bitnami/readme-generator-for-helm

👍 
will do it in a standalone PR

**Review by viceice — APPROVED** [review]
I hope you tested the indention changes

**Review by PhilipAbed — APPROVED** [review]

File: helm-charts/mend-renovate-ce/templates/deployment.yaml [hunk 5]
```diff
@@ -212,7 +261,11 @@ spec:
           {{- end }}
       {{- with .Values.nodeSelector }}
       nodeSelector:
-        {{- toYaml . | nindent 8 }}
+      {{- toYaml . | nindent 8 }}
+      {{- if $.Values.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ $.Values.imagePullSecrets }}
+      {{- end }}
```
Comment (viceice) [inline helm-charts/mend-renovate-ce/templates/deployment.yaml:268]:
```suggestion
      {{- end }}
      {{- if .Values.imagePullSecrets }}
      imagePullSecrets:
        - name: {{ .Values.imagePullSecrets }}
```

**Review by viceice — CHANGES_REQUESTED** [review]

File: helm-charts/mend-renovate-ee/templates/server-deployment.yaml [hunk 6]
```diff
@@ -183,7 +223,11 @@ spec:
           {{- end }}
       {{- with .Values.renovateServer.nodeSelector }}
       nodeSelector:
-        {{- toYaml . | nindent 8 }}
+      {{- toYaml . | nindent 8 }}
+      {{- if .Values.renovateServer.imagePullSecrets }}
```
Comment (viceice) [inline helm-charts/mend-renovate-ee/templates/server-deployment.yaml:227]: same here

File: helm-charts/mend-renovate-ee/templates/worker-deployment.yaml [hunk 7]
```diff
@@ -135,7 +123,11 @@ spec:
           {{- end }}
       {{- with .Values.renovateWorker.nodeSelector }}
       nodeSelector:
-        {{- toYaml . | nindent 8 }}
+      {{- toYaml . | nindent 8 }}
+      {{- if .Values.renovateWorker.imagePullSecrets }}
```
Comment (viceice) [inline helm-charts/mend-renovate-ee/templates/worker-deployment.yaml:127]: same here

**Review by viceice — CHANGES_REQUESTED** [review]
@nabeelsaabna sorry for not seeing this before

**Comment by viceice** [comment]
- #472

**Pull request merged**